src/Schema/Events/Toggle.ts

```diff
@@ -35,3 +35,31 @@ export interface ToggledNotification {
   original: number
   subject: PushNotificationType
 }
+
+/**
+ * A user toggles a saved search on/off on iOS
+ *
+ *  This schema describes events sent to Segment from [[toggledNotification]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "toggledSavedSearch",
+ *    context_screen_owner_type: "artist",
+ *    context_screen_owner_id: "58de681f275b2464fcdde097",
+ *    context_screen_owner_slug: "anthony-hunter",
+ *    modified: 1,
+ *    original: 0,
+ *    saved_search_id: "58de681f275b2464fcdde097"
+ *  }
+ * ```
+ */
+export interface ToggledSavedSearch {
+  action: ActionType.toggledSavedSearch
+  context_screen_owner_type: ScreenOwnerType
+  context_screen_owner_id?: string
+  context_screen_owner_slug?: string
+  modified: number
+  original: number
+  saved_search_id: string
```

Comment:
We have two models in the database `SearchCriteria` and `UserSearchCriteria`. The `SearchCriteria` model holds the interesting bits, including the criteria attributes. `UserSearchCriteria` is the "join" model which exists to associate a user with a particular search criteria.

In the app, we have access to the `search_criteria_id`. If that's what we want to track here, what do you think of updating this attribute name to match?

Reply:
I think this goes back to [this](https://artsyproduct.atlassian.net/browse/FX-2808): the definition of the Saved Search model. From an analytics and domain model data standpoint we would need 1 object which holds the user saved search info and we would use the identifier here. I think we should resolve FX-2808 before proceeding here.

Reply:
Going back to this after our [slack conversation](https://artsy.slack.com/archives/C9SATFLUU/p1623862368096700). Ideally we would use the  `UserSearchCriteria` identifier for `saved_search_id`. I want to directly identify the saved search user toggled. 
However, given that it seems we can only access the `search_criteria_id` in the app, and given all users are logged in in the app, then we should be able to retrieve the `UserSearchCriteria` object by using the `search_criteria_id` and the `user_id`. So seems it is fine to use the `search_criteria_id`!
Thank you @dblandin